packages/ui/src/kit/VTabs/VTabs.vue

```diff
@@ -1,27 +1,37 @@
 <template>
-  <div :class="vTabsCssClass">
+  <div
+    v-if="$slots.default"
+    :class="vTabsCssClass"
+  >
     <section class="v-tabs-controls-list">
       <button
-        v-for="(child, index) in $slots.default.filter($el => !!$el.tag)"
+        v-for="(tab, index) in $slots.default.filter($el => !!$el.tag)"
         :key="index"
         :class="{
           'v-tabs-control': true,
-          'is-active': activeTabId === getChildProps(child).id,
+          'is-active': activeTabIdx === index,
         }"
-        v-bind="child.data.attrs"
-        @click="onTabClick($event, index)"
+        v-bind="tab.data.attrs"
+        @click="onTabClick({ event: $event, index })"
       >
-        {{ getChildProps(child).label }}
+        {{ getTabLabel(tab) }}
       </button>
     </section>
-    <section class="v-tabs-content">
+    <div v-show="false">
```

Comment:
No, we need to have access to all tabs before mounting, because buttons rendering based on each tab. If we will remove that we can't render tabs buttons as is.

Reply:
aah, i understand, doh...
maybe add mark for this? it's not obvios, when looked at first time, like comment or something else?

Reply:
Only at comment level, I guess 🤷‍♀